src/align.py

```diff
@@ -16,7 +20,30 @@ def align(x: str, y: str, edits: str) -> tuple[str, str]:
     ('ACCACAGT-CATA', 'A-CAGAGTACAAA')
 
     """
-    return "", ""
+    x_align = ''
+    y_align = ''
+    i = 0
+    j = 0
+
+    for _ in edits:
```

Comment:
I wouldn't use `_` as a standard variable. Traditionally, it is used for a value we don't care about (and that goes for multiple languages, although it isn't formalised in Python).